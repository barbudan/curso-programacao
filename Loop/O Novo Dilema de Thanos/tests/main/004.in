8
1325745
1872387
18128
736
1
2
3
1000000000008