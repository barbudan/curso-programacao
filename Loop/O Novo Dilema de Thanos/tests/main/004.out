Thanos poupara o Planeta 1.
Thanos poupara o Planeta 2.
Thanos nao poupara o Planeta 3.
Thanos nao poupara o Planeta 4.
Thanos poupara o Planeta 5.
Thanos nao poupara o Planeta 6.
Thanos poupara o Planeta 7.
Thanos nao poupara o Planeta 8.
